app/src/main/java/com/awscherb/cardkeeper/data/handler/BaseRealmHandler.java

```diff
@@ -4,67 +4,50 @@
 
 import java.util.List;
 
+import io.reactivex.Completable;
+import io.reactivex.Observable;
```

Comment:
I think `Observable` is the correct choice to use based on number of elements in the flow, referencing https://github.com/ReactiveX/RxJava/wiki/What%27s-different-in-2.0#observable-and-flowable